### PR TITLE
update products when sales is disabled

### PR DESCRIPTION
**Pull request by kascote**
@ericsaupe, @jeffreyfuchang

### CODE REVIEW

**Comment by kascote** [comment]
@ericsaupe better ?

**Comment by ericsaupe** [comment]
Yup. Thanks

File: lib/tasks/update_sales_documents.rake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+namespace :spree_sale_pricing do
+
+  desc "Update product documents when SalePrice is disabled"
+  task :update_product_documents => :environment do
+
+    sale_prices = Spree::SalePrice.where('(end_at >= ?) AND (end_at <= ?)', Time.now - 1.day, Time.now)
```
Comment (jeffreyfuchang) [inline lib/tasks/update_sales_documents.rake]: Will need to make the Time.now with TimeWithZone 

```
Spree::SalePrice.last.end_at.class
=> ActiveSupport::TimeWithZone < Object
```

**Comment by ericsaupe** [comment]
closing because of #28

**Pull request closed without merging**